ddl.dic

```diff
@@ -112,6 +116,25 @@ save_alias.dictionary_uri
 
 save_
 
+save_alias.dictionary_version
+
+    _definition.id                '_alias.dictionary_version'
+    _definition.class             Attribute
+    _definition.update            2024-04-04
+    _description.text
+;
+    A version identifier of the dictionary that includes the latest compatible
```

Comment:
This still reads to me like we are asking for the latest version of the dictionary. I prefer "The version identifier of a dictionary containing a compatible version of the aliased definition."

Otherwise as dictionary versions increase with no change in an aliased definition, we are supposed to update our alias version to maintain conformance. Better to stick a flag in at some point in that dictionary's development, and the user can navigate forward and back to see if/when the definition changes.

Reply:
What I meant here is that the version number should point to any version which contains the latest compatible definition (e.g. if the last compatible change occurred in version 0.1.0, but there were also version 0.2.0, 0.2.1, etc., we can point to any of those). However, you are correct to say, that this would still require us to track the changes in the external dictionaries so I reworded the definition as you suggested.

However, I did write "of the dictionary" instead of "of a dictionary" since the version number is tied to a specific dictionary given in `_alias.dictionary_uri`. Feel free to change it to "a dictionary" (or suggest a change) if that is not grammatically correct.

If you are happy with the current definition, please mark the issue as resolved.